README.md

```diff
@@ -5,11 +5,11 @@
 ## 開発／動作環境(develop/run-time environment)
```

Comment:
@daisukenishino 

Please divide this file into *English Version* and *Japanese Version*.
(Please see [this](https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/README.md).)